### ISSUE TITLE
Unable to open database file

### ISSUE THREAD

**Issue by luizperes** [naturalness/sensibility#4]
Sensibility emits "(sqlite3.OperationalError) unable to open database file" when user chooses `javascript` after cloning project

```
(sensibility) Luizs-MacBook-Pro:sensibility luizperes$ sensibility mine download
INFO:ConcreteLanguageProxy:Language inferred from environment: javascript
Traceback (most recent call last):
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2147, in _wrap_pool_connect
    return fn()
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 328, in unique_connection
    return _ConnectionFairy._checkout(self)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 766, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 516, in checkout
    rec = pool._do_get()
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 1229, in _do_get
    return self._create_connection()
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 333, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 461, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 651, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 105, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 393, in connect
    return self.dbapi.connect(*cargs, **cparams)
sqlite3.OperationalError: unable to open database file

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/luizperes/Projects/sensibility/bin/mine/download", line 35, in <module>
    downloader = Downloader()
  File "/Users/luizperes/Projects/sensibility/sensibility/miner/downloader.py", line 55, in __init__
    self.corpus = Corpus(writable=True)
  File "/Users/luizperes/Projects/sensibility/sensibility/miner/corpus.py", line 129, in __init__
    if self.empty:
  File "/Users/luizperes/Projects/sensibility/sensibility/miner/corpus.py", line 153, in empty
    metadata.reflect(self.engine)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/sql/schema.py", line 3791, in reflect
    with bind.connect() as conn:
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2091, in connect
    return self._connection_cls(self, **kwargs)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 90, in __init__
    if connection is not None else engine.raw_connection()
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2177, in raw_connection
    self.pool.unique_connection, _connection)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2151, in _wrap_pool_connect
    e, dialect, self)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1465, in _handle_dbapi_exception_noconnection
    exc_info
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 186, in reraise
    raise value.with_traceback(tb)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2147, in _wrap_pool_connect
    return fn()
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 328, in unique_connection
    return _ConnectionFairy._checkout(self)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 766, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 516, in checkout
    rec = pool._do_get()
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 1229, in _do_get
    return self._create_connection()
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 333, in _create_connection
    return _ConnectionRecord(self)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 461, in __init__
    self.__connect(first_connect_check=True)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/pool.py", line 651, in __connect
    connection = pool._invoke_creator(self)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py", line 105, in connect
    return dialect.connect(*cargs, **cparams)
  File "/Users/luizperes/Projects/sensibility/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 393, in connect
    return self.dbapi.connect(*cargs, **cparams)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
```

**Comment by eddieantonio** [comment]
Fixed in #5.

**Comment by eddieantonio** [comment]
Reverted #8; must fix again.
